tests/system/source/main.d

```diff
@@ -39,11 +40,16 @@ immutable string PREFIX = ">>CI:";
 
 int main (string[] args)
 {
-    writefln("%s START SYSTEM TEST", PREFIX);
+    void log (string message)
+    {
+        writefln("%s [%s] %s", PREFIX, Clock.currTime().toISOString , message);
+    }
```

Comment:
Move it to module level. But we already have timestamps in the CI. I guess that's for local testing ?

Reply:
I think I can just drop the commit that adds this and the logging config. When testing locally I was not using the defaults and the time stamps were only to verify the polling was actually at the expected frequency.